Documentation/network/bgp-control-plane.rst

```diff
@@ -382,6 +384,30 @@ When the service has ``externalTrafficPolicy: Local``, ``BGP Control Plane`` kee
 of the endpoints for the service on the local node and stops advertisement when there's
 no local endpoint.
 
+CiliumPodIPPool announcements
+-----------------------------
+
+By default, virtual routers will not announce CiliumPodIPPool CIDRs. Virtual routers will announce
```

Comment:
```suggestion
By default, virtual routers will not announce all CiliumPodIPPool CIDRs. Virtual routers will announce
```
small clarification

Reply:
> By default, virtual routers will not announce all CiliumPodIPPool CIDRs.

`all` could imply that `some` CiliumPodIPPool CIDRs will be advertised. I'll update the PR to use `any`.

Reply:
Commit 77abf51 resolves this comment.

Reply:
Sorry @danehans, I don't understand this paragraph :sweat_smile: These two sentences seem contradictory to me:
```
By default, virtual routers will not announce any CiliumPodIPPool CIDRs.

Virtual routers will announce allocated CIDRs of any CiliumPodIPPool that matches the ``.podIPPoolSelector`` of the virtual router.
```

So the default behaviour is to not announce any CIDRs or to announce some CIDRs (matching the selector)?

Sorry I'm so picky here :) Just not knowing much about Cilium BGP things I'm a bit confused by the docs.

Reply:
@lambdanis I updated the doc in commit df33092. Hopefully, this resolves your confusion. Thanks for the review.

Reply:
Thank you! It's clear to me now.